site/app/templates/admin/admin_gradeable/AdminGradeableCreate.twig

```diff
@@ -533,6 +533,26 @@
         //
         // Gradeable Type Radio Button Events
         //
+        
+        $("input[type='radio']").change(validate);
+        
+        function validate() 
+        {
```

Comment:
The curly bracket for JavaScript files should be on the same line as function declarations and if states and code blocks in general. (You can take a look at some of the standalone files under site/public/js or in the rest of this file